news/_posts/2023-09-29-plugin-v2.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "ilastik Fiji plugin version 2 released"
+---
+
+We are happy to announce the release of version 2.0 of our Fiji plugin.
+The plugin allows you to conveniently import and export image in HDF5 format, as well as running trained Pixel Classification, Autocontext, Object Classification, Multicut, and Tracking Workflows from within Fiji.
+
+
+HDF5 is a format that allows ilastik efficient block-wise processing of data.
+You will get the best performance in ilastik using this format.
+
+The plugin was almost completely rewritten by Emil Melnikov from our team.
+In this new plugin version the reading speed for HDF5 files into Fiji has been improved 100x (!) compared to the previous version.
+Writing is roughly 3x faster, too!
+We took care to keep the plugin backwards compatible - all your macros should run as before.
```

Comment:
We could also say here that the Java API has changed, but I'm not sure if this is worth mentioning.